electrum/wallet.py

```diff
@@ -1837,6 +1837,37 @@ def get_new_sweep_address_for_channel(self) -> str:
         assert is_address(selected_addr), f"not valid bitcoin address: {selected_addr}"
         return selected_addr
 
+    def should_keep_reserve_utxo(
+            self,
+            tx_inputs: List[PartialTxInput],
+            tx_outputs: List[PartialTxOutput],
+            is_anchor_channel_opening: bool,
+    ) -> bool:
```

Comment:
I would like to suggest making most parameters in most functions kw-only in general, except maybe the first one or two parameters that are trivially deducible just from the function name.
```suggestion
    def should_keep_reserve_utxo(
            self,
            *,
            tx_inputs: List[PartialTxInput],
            tx_outputs: List[PartialTxOutput],
            is_anchor_channel_opening: bool,
    ) -> bool:
```